requirements.txt

```diff
@@ -9,6 +9,7 @@ jinja2>=2.10
 jsonpatch>=1.22
 jsonschema>=2.5.1
 makefun>=1.7.0,<2
+marshmallow==3.7.1
```

Comment:
This is the version we vendored - looks like Cloud uses `3.8.0`. Should we match them?

Reply:
I think so